normalize_and_clean_data.py

```diff
@@ -196,8 +199,33 @@ def main() -> None:
     df = replace_yes_no(df)
     df = replace_ownership_value(df)
     check_missing_values(df)
-    save_clean_data(df, clean_folder)
+
+    save_cleaned_data(df, file_path, save_directory)
+    return True
+
+def main() -> None:
+    logging.info('-----Start-----')
+    base_directory: str = r'.\split_by_type_year_data'
+    clean_folder: str = r'.\clean_data'
+    
+    processed_files = []
+
+    for root, _, files in os.walk(base_directory):
+        for file in files:
+            if file.endswith('.csv'):
+                file_path = os.path.join(root, file)
+                result = process_file(file_path, clean_folder)
+                processed_files.append((file_path, result))
+
+    logging.info('----Processed info: ')
+    for file_path, result in processed_files:
+        
```

Comment:
Extra space delete it.
Try **[autopep8](https://pypi.org/project/autopep8/)**, **[black formmater](https://pypi.org/project/black/)**, **[isort](https://pycqa.github.io/isort/)** packages to automatic syntaxes check.

Reply:
+